kong/runloop/handler.lua

```diff
@@ -1419,6 +1424,16 @@ return {
       if var.http_proxy_connection then
         clear_header("Proxy-Connection")
       end
+
+      -- X-Kong-Request-Id upstream header
+      local rid, rid_get_err = request_id.get()
```

Comment:
Suggest all calls to `request_id.get` be cached to `request_id_get()`, hot codepath function.

Reply:
done, thanks